sashimi-webapp/src/App.vue

```diff
@@ -148,7 +148,7 @@ a {
     }
 
     img {
-      width: 40px;
+      width: 150px;
```

Comment:
Un-prefixed CSS are used here for common HTML tag. It will affect the entire webapp.
However, if this is intentional, then it is fine.

Reason being, in the coming version, we will put all the viewer's content inside an iframe. Therefore, this css will not be able to pollute the content in iframe anymore.